sample_files/site_theme/override.css

```diff
@@ -8,8 +8,8 @@ body {
     background: rgba(240,240,240, 0.85) !important;
 }
 
-/* Force sidebars to 220px to line up with the image in top_bar.html */
-#container #left_sidebar, #container #right_sidebar {
+/* Force sidebars to 220px to line up with the image */
+#container {
```

Comment:
This is not a safe change.  The `#container` selector alone selects the element with the ID `container`, while the previous `#container #left_sidebar` combinator selected the ID `left_sidebar` which was a descendant of the `container` ID.

Given that you've removed the `left_sidebar` and `right_sidebar` IDs, you can safely just remove this entire chunk of CSS.

Reply:
I dont think I understand why  its not safe but I do get that if I deleted all the other stuff than I should've deleted this to. Removed the whole container chunk of code.

Reply:
Also I made the change to Global footer 35-44.